js/src/lib/Details/Markdown.js

```diff
@@ -87,7 +86,12 @@ const renderAndEscapeMarkdown = ({ source, githubRepo }) => {
     return `<pre><code>${code}</code></pre>`;
   };
 
-  return marked(source, { renderer, mangle: false, sanitize: true });
+  const html = marked(source, { renderer, mangle: false });
+  return xss(html, {
+    whiteList: Object.assign({}, xss.getDefaultWhiteList(), {
```

Comment:
you can use spreading here if you want, it's supported by our babel config 😄

Reply:
Thanks! just changed.